jekyll/_cci2/language-python.md

```diff
@@ -2,104 +2,266 @@
+## Configuration Walkthrough
 
-In the project you will find a commented CircleCI configuration file <a href="https://github.com/CircleCI-Public/circleci-demo-python-django/blob/master/.circleci/config.yml" target="_blank">`.circleci/config.yml`</a>. This file shows best practice for using CircleCI 2.0 with Python projects.
+Every CircleCI project requires a configuration file called `.circleci/config.yml`.
+Follow the steps below
+to create a complete `config.yml` file.
 
-Refer to the [Project Walkthrough]({{ site.baseurl }}/2.0/project-walkthrough/) for a sample Flask application: <https://github.com/CircleCI-Public/circleci-demo-python-flask>.
+### Specify a Version
 
-## Pre-Built CircleCI Docker Images
+Every `config.yml` starts with the [`version`]({{ site.baseurl }}/2.0/configuration-reference/#version) key.
+This key is used
+to issue warnings about breaking changes.
 
-We recommend using a CircleCI pre-built image that comes pre-installed with tools that are useful in a CI environment. You can select the Python version you need from Docker Hub: <https://hub.docker.com/r/circleci/python/>. The demo project uses an official CircleCI image.
+```yaml
+version: 2
+```
+
+### Create a Build Job
 
-Database images for use as a secondary 'service' container are also available on Docker Hub in the `circleci` directory.
+A run is comprised of one or more [jobs]({{ site.baseurl }}/2.0/configuration-reference/#jobs).
+Because this run does not use [workflows]({{ site.baseurl }}/2.0/configuration-reference/#workflows),
+it must have a `build` job.
 
-## Build the Demo Django Project 
+Use the [`working_directory`]({{ site.baseurl }}/2.0/configuration-reference/#job_name) key
+to specify where a job's [`steps`]({{ site.baseurl }}/2.0/configuration-reference/#steps) run.
 
-A good way to start using CircleCI is to build a project yourself. Here's how to build the demo project with your own account:
+```yaml
+version: 2
+jobs:
+  build:  # required for runs that don't use workflows
+    working_directory: ~/circleci-demo-python-django
+```
 
-1. Fork the project on GitHub to your own account.
-2. Go to the [Add Projects](https://circleci.com/add-projects){:rel="nofollow"} page in CircleCI and click the Build Project button next to the project you just forked.
-3. To make changes you can edit the `.circleci/config.yml` file and make a commit. When you push a commit to GitHub, CircleCI will build and test the project.
+### Choose an Executor Type
 
-## Config Walkthrough
+A job's steps occur in a virtual environment called an [executor]({{ site.baseurl }}/2.0/executor-types/).
 
-Start with the version:
+In this example,
+the [`docker`]({{ site.baseurl }}/2.0/configuration-reference/#docker) executor is used
+to specify a custom Docker image.
+The first image listed becomes the job's [primary container]({{ site.baseurl }}/2.0/glossary/#primary-container).
+All of a job's commands execute in this container.
```

Comment:
no possessive for objects, just say  'becomes the primary container for the job run' or something and 'commands execute in this container'.